packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -101,14 +104,19 @@ export function PanelChrome({
   const containerStyles: CSSProperties = { width, height };
   const ariaLabel = title ? selectors.components.Panels.Panel.containerByTitle(title) : 'Panel';
 
+  // Shift the hover menu down if it's on the top row so it doesn't get clipped by topnav
+  const yOffset = (gridPos?.y ?? 0) === 0 ? -16 : undefined;
```

Comment:
Imo, in general the responsibility of `PanelChrome` should not be to make calculations based on a `gridPos`. If it is used anywhere outside of a dashboard it might make no sense/use to the dev why exactly -16 px.
Maybe the `PanelStateWrapper` should calculate the y-offset and send to `PanelChrome` only a prop that references the hovering header's position directly. Something like a prop `yOffsetHoverHeader={-16}` ?

Reply:
I agree with @polibb, if we can do these calculations in the `PanelStateWrapper` it will be better, especially when deprecating props in grafana ui is quite painful.

Reply:
yea, agree that it's probably better to just pass in a hoverHeaderOffset or something